doc/source/index.rst

```diff
@@ -19,194 +19,9 @@ They ensure the PyMAPDL accuracy.
 Summary of the exposed Verification Manual Examples
 ---------------------------------------------------
 
-This section demonstrates a range of Mechanical APDL elements and
-capabilities in straightforward problems which have "classical" or
-readily-obtainable theoretical solutions using PyMAPDL.
-
-These problems may then serve as the basis for additional validation
-and qualification of Mechanical APDL capabilities for specific
-applications that may be of interest.
-
-
-
-.. raw:: html
-
-    <div class="sphx-glr-thumbnails">
-
-
-.. raw:: html
-
-    <div class="sphx-glr-thumbcontainer" tooltip="Reference:  - S. Timoshenko, Strength of Materials, Part I, Elementary Theory and    Problems, ...">
```

Comment:
This could be added back if you want. Or in a follow up PR. :)

Reply:
Although I dont really like to add so much ``raw`` and ``only:: html`` statements.